static/css/frame.scss

```diff
@@ -40,48 +48,62 @@
 
 .shot-main-actions {
 
-  @include respond-to("medium") {
+  @include respond-to("large") {
     @include flex-container(row, flex-start, center);
-    margin-right: 10px;
+    flex: initial;
     padding-top: 4px;
     overflow: hidden;
   }
 
   @include respond-to("small") {
     @include flex-container(row, flex-start, center);
     background: $light-default;
-    border-bottom: 1px solid rgba(#000, 0.1);
-    border-top: 1px solid $light-border;
-    flex: 0 0 100%;
-    height: $grid-unit * 3.6;
-    order: 1;
-    width: 100%;
+    height: $grid-unit * 3.5;
   }
 
   flex: 1;
   position: relative;
 
-  .expire-widget {
+  .expire-info {
+    display: block;
     @include respond-to("small") {
       display: none;
     }
+    @include respond-to("large") {
+      display: none;
+    }
+    @media (min-width: 1260px) {
```

Comment:
Let's stick with the `respond-to($size)` pattern, instead of using a media query directly. You can call this "extra-large".

Reply:
Done